routing/turns_tts_text.cpp

```diff
@@ -7,6 +7,9 @@
 #include <iterator>
 #include <string>
 #include <utility>
+#include <codecvt>
+#include <locale>
+#include <cstdio>
```

Comment:
1. Please sort includes. 
2. I think they are not needed at all. See other comments below.